assets/scss/_custom.scss

```diff
@@ -806,28 +806,31 @@ body.td-documentation {
 @media (max-width: 768px) {
   #announcement {
     padding-top: 4rem;
-    padding-bottom: 4rem;
-    .announcement-main, aside .announcement-main {
-      padding-top: calc(min(2rem,2em));
-    }
-  }
-}
+    padding-bottom: 0;
+    h4{
+      display: flex;
+      flex-direction: column;
+      text-align: center;
 
-@media (max-width: 480px) {
-  #announcement {
-    padding-bottom: 0.5em;
-  }
-  #announcement aside {
-    h1, h2, h3, h4, h5, h6 {
-      img.event-logo {
-        margin-left: auto;
-        margin-right: auto;
-        margin-bottom: 0.75em;
-        display: block;
-        max-height: initial;
-        max-width: calc(min(calc(100vw - 2em), 240px));
-        float: initial;
+      a {
+        font-weight: bold;
       }
+      
+      row-gap: 1rem;
+      font-size: 1.5em;
+
+      img {
+        order: 2;
```

Comment:
How do we know this value is correct?

Reply:
I see the problem here, I'll figure something out